packages/ckeditor5-dev-release-tools/lib/tasks/reassignnpmtags.js

```diff
@@ -43,10 +45,11 @@ module.exports = async function reassignNpmTags( { npmOwner, version, packages }
 		if ( latestVersion === version ) {
```

Comment:
In my opinion `npm show` call is not needed. It increases the execution time, and decreases the code readability.

Reply:
After discussing this topic on devops daily, we decided to remove `npm show` call, because:
1. This logic is built in in the second call: `npm dist-tag add`. If a tag already exists it is not updated.
2. Using both `npm show` and `npm dist-tag add` increases the execution time. (2 calls instead of 1).
3. Error handling could be confusing with 2 calls. We would not be sure if the displayed error refers to `npm show` or `npm dist-tag add`.